src/RootFS.jl

```diff
@@ -189,22 +191,24 @@ function update_rootfs(triplets::Vector{S}; automatic::Bool = automatic_apple,
                 verbose = verbose,
                 force = true
             ) && file_existed)
-                # Unmount the mountpoint. It may point to a previous version
-                # of the file. Also, we're about to mess with it
-                if success(`mountpoint $(dest_dir)`)
-                    run(`sudo umount $(dest_dir)`)
+                if Compat.Sys.islinux()
+                    # Unmount the mountpoint. It may point to a previous version
+                    # of the file. Also, we're about to mess with it
+                    if success(`mountpoint $(dest_dir)`)
+                        run(`sudo umount $(dest_dir)`)
+                    end
+
+                    # Patch squashfs files for current uid
+                    rewrite_squashfs_uids(squashfs_path, ccall(:getuid, Cint, ()))
+
+                    # Touch SHA file to prevent re-verifcation from blowing the
+                    # fs away
+                    touch(string(squashfs_path,".sha256"))
                 end
-
-                # Patch squashfs files for current uid
-                rewrite_squashfs_uids(squashfs_path, ccall(:getuid, Cint, ()))
-                
-                # Touch SHA file to prevent re-verifcation from blowing the
-                # fs away
-                touch(string(squashfs_path,".sha256"))
             end
-            
+
             # Then mount it, if it hasn't already been mounted:
-            if !success(`mountpoint $(dest_dir)`)
+            if Compat.Sys.islinux() && !success(`mountpoint $(dest_dir)`)
                 mkpath(dest_dir)
                 run(`sudo mount $(squashfs_path) $(dest_dir) -o ro,loop`)
             end
```

Comment:
Should the `sudo umount` a few lines below this (on line 217) also be gated on `islinux()`?

Reply:
yes